platform/ios/CHANGELOG.md

```diff
@@ -2,10 +2,7 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## master
```

Comment:
We should keep this section here on the `master` branch. So long as this section doesn’t exist in `release-frappé`, cherry-picking this commit there should go smoothly.